src/TopDownProteomics/ProForma/TopPicProformaParser.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace TopDownProteomics.ProForma
+{
+    /// <summary>
+    /// A parser for TopPic strings into a ProformaTerm <see cref="ProFormaTerm"/>
```

Comment:
The name is TopPIC, not TopPic ... not sure if we should change the class name, but comments should case it right

Reply:
Left the class name for now